compiler/record-hessian/src/HessianObserver.cpp

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
plz split this class to another PR.
let's make codes shorter for better review.